src/log.c

```diff
@@ -70,7 +70,10 @@ static unsigned int logging_initialized = FALSE;     /* boolean */
  */
 int open_log_file (const char *log_file_name)
 {
-        log_file_fd = create_file_safely (log_file_name, FALSE);
+	if (config.godaemon == FALSE)
```

Comment:
These lines use tabs instead of spaces.

I personally prefer tabs, but this project uses spaces.
So let's stay consistent for now?!

Reply:
well, there are some lines here already that use tabs and some that use spaces - not my fault :)

Reply:
Uhm. Best I can tell, there were no tabs in the context of your patch. And the patch adds four lines with spaces instead of tabs.